packages/react-jss/src/JssProvider.js

```diff
@@ -20,85 +20,88 @@ type Props = {
   generateId?: GenerateId,
   classNamePrefix?: string,
   disableStylesGeneration?: boolean,
+  sheetOptions: StyleSheetFactoryOptions,
   children: Node
 }
 
 export default class JssProvider extends Component<Props> {
   static propTypes = {
-    ...propTypes,
+    registry: PropTypes.instanceOf(SheetsRegistry),
+    jss: PropTypes.shape({}),
     generateId: PropTypes.func,
     classNamePrefix: PropTypes.string,
     disableStylesGeneration: PropTypes.bool,
-    children: PropTypes.node.isRequired
+    children: PropTypes.node.isRequired,
+    sheetOptions: PropTypes.shape({})
   }
 
-  static childContextTypes = contextTypes
+  managers: {} = {}
 
-  static contextTypes = contextTypes
+  generateId = createGenerateId()
 
-  // JssProvider can be nested. We allow to overwrite any context prop at any level.
```

Comment:
So we don't support overriding a specific option any more from a particular provider, right? I think this is a breaking change. We need to document etc.

Reply:
We do.